spec/inputs/http_spec.rb

```diff
@@ -6,20 +6,25 @@
 
 describe LogStash::Inputs::Http do
 
+  before do
```

Comment:
not related to this specs, but I guess on the feature we should make this standard in our helpers, I've used this same pattern so often for now.